detect/fixeddetector.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from .detector import Detector
+from dataset import Table
+
+class FixedDetector(Detector):
+    def __init__(self, name, src, fpath=None, 
+                df=None, db_conn=None, table_query=None, db_engine=None):
+        super(FixedDetector, self).__init__(name)
+        self._errors_table = Table(name, src, na_values=None, 
+            exclude_attr_cols=[], fpath=fpath, df=df, db_conn=db_conn, 
+            table_query=table_query, db_engine=db_engine)
+        if set(list(self._errors_table.df)) != set(['_tid_', 'attribute']):
+            raise Exception('Invalid FixedDetector table for %s' % name)
```

Comment:
I am not sure what this message means... The file is invalid, not the table.

Reply:
Changed the message to include the file as the cause.